docs/csharp/programming-guide/types/how-to-convert-between-hexadecimal-strings-and-numeric-types.md

```diff
@@ -55,6 +55,12 @@ These examples show you how to perform the following tasks:
   
  [!code-csharp[csProgGuideTypes#38](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsProgGuideTypes/CS/Class1.cs#38)]  
   
+## Example  
+
+ The following example shows how to convert a [byte](../../language-reference/builtin-types/integral-numeric-types.md) array to a hexadecimal string by calling the <xref:System.Convert.ToHexString?displayProperty=nameWithType> method.  
```

Comment:
```suggestion
 The following example shows how to convert a [byte](../../language-reference/builtin-types/integral-numeric-types.md) array to a hexadecimal string by calling the <xref:System.Convert.ToHexString?displayProperty=nameWithType> method introduced in .NET 5.0.
```

Reply:
Can you apply this suggestion? It will fix a build warning, and adds the version information for readers.